lib/inc/internal/facts/linux/os_osrelease.hpp

```diff
@@ -81,7 +82,17 @@ namespace facter { namespace facts { namespace linux {
         virtual std::string get_release(std::string const& name, std::string const& distro_release) const override
         {
             auto val = _release_info.find("VERSION_ID");
-            return (val != _release_info.end()) ? val->second : std::string();
+            if (val != _release_info.end()) {
+                string major, minor;
```

Comment:
These two variables look unused - otherwise this makes sense and looks fine to me

Reply:
Ah, and I believe this should be targeted at 3.11.x for agent 5.5.x

Reply:
Nice catch, Casey.